hummingbird/ml/operator_converters/__init__.py

```diff
@@ -47,6 +47,7 @@
 __pdoc__ = {}
 __pdoc__["hummingbird.operator_converters._array_feature_extractor_implementations"] = True
 __pdoc__["hummingbird.operator_converters._decomposition_implementations"] = True
+__pdoc__["hummingbird.operator_converters._discretizer_implementations`"] = True
```

Comment:
I think you have an extra tick-mark at the end

Reply:
fixed.